crates/astria-sequencer-relayer/src/config/mod.rs

```diff
@@ -14,7 +14,11 @@ use serde::{
 
 mod cli;
 
-const DEFAULT_BLOCK_TIME: u64 = 1000;
+/// Max time in ms to wait for a block to finalize after it is received from the sequencer.
```

Comment:
what happens if it does not finalize in that time? also this appears to only be used in tests, can you move it to that module?

Reply:
obsolete